testsuite/features/support/constants.rb

```diff
@@ -1543,4 +1543,4 @@
   'uyuni-proxy-devel-leap-x86_64' => 60
 }.freeze
 
-EMPTY_CHANNELS = %w[sle-module-suse-manager-retail-branch-server-4.3-updates-x86_64].freeze
+EMPTY_CHANNELS = %w[sle-module-suse-manager-retail-branch-server-4.3-updates-x86_64 sle-manager-tools15-beta-pool-x86_64-sp4 fake-base-channel-suse-like fake-base-channel-i586 test-base-channel-x86_64].freeze
```

Comment:
Is it going to looks lengthy, maybe something like:
```suggestion
EMPTY_CHANNELS = [
  "sle-module-suse-manager-retail-branch-server-4.3-updates-x86_64",
  "sle-manager-tools15-beta-pool-x86_64-sp4",
  "fake-base-channel-suse-like",
  "fake-base-channel-i586",
  "test-base-channel-x86_64"
].freeze
```

Reply:
Done. Without double quotes, but done :)